storage/imxrt-flash/flashsrv.c

```diff
@@ -492,6 +501,9 @@ static void flashsrv_rawThread(void *arg)
 		if (msg.oid.id >= memory->pCnt) {
 			msg.o.err = -EINVAL;
 		}
+		else if (!(memory->parts[msg.oid.id].authProcPid == 0 || memory->parts[msg.oid.id].authProcPid == msg.pid)) {
+			msg.o.err = -EACCES;
```

Comment:
Blocking gettatr can cause that ls will stop work

Reply:
not sure, this is gettatr for the device, getattr for filesystem should be taken care of by dummyfs?

Still, don't use `!` if not needed - makes it harder to understand, eg:
`if (memory->parts[msg.oid.id].authProcPid != 0 && memory->parts[msg.oid.id].authProcPid != msg.pid) return -EACCES`

Reply:
Hmm, right. This getattr is used for seek after opening file. Sorry, my mistake..

Reply:
> Still, don't use `!` if not needed - makes it harder to understand, eg: `if (memory->parts[msg.oid.id].authProcPid != 0 && memory->parts[msg.oid.id].authProcPid != msg.pid) return -EACCES`

Hmm, my perspective was that it can be read as `!hasAccess` and it's easier to undarstand. Maybe now it's clear?